lib/Models/CatalogItem.js

```diff
@@ -334,6 +354,8 @@ CatalogItem.defaultPropertiesForSharing.push('isEnabled');
 CatalogItem.defaultPropertiesForSharing.push('isShown');
 CatalogItem.defaultPropertiesForSharing.push('isLegendVisible');
 CatalogItem.defaultPropertiesForSharing.push('nowViewingIndex');
+CatalogItem.defaultPropertiesForSharing.push('isMappable');
+CatalogItem.defaultPropertiesForSharing.push('showsInfo');
```

Comment:
These two properties don't need to be included in share URLs, since they're usually configured from a catalog (as opposed to a user-modified value like `isShown`).